main.py

```diff
@@ -1,88 +1,18 @@
-import pygame as pg 
-import os
-import time
-import random
-from const import *
-from player import Player
-from challenge1 import Challenge1
-from lec1 import Lecture11, Lecture12
+from lib.game import Lesson1Game
 
-#create screen
-WIN = pg.display.set_mode((WIDTH,HEIGHT))
-pg.display.set_caption("CS FOUNDATION")
 
-#main function
-def main():
-    global main_background_state
-    main_background_state = IDLE_STATE
+def answer_what_is_your_name():
```

Comment:
Students will put their solution here. We need to add a bunch of comments to instruct them on how to do it.